onnxruntime/core/providers/cpu/tensor/slice.cc

```diff
@@ -82,59 +111,160 @@ Status SliceBase::PrepareForCompute(const std::vector<int64_t>& raw_starts,
   }
 
   // Iterate through the provided axes and override the start/end ranges
+  std::unordered_set<int64_t> unique_axes;
   const auto& dimension_count = input_dimensions.size();
   for (size_t axesIndex = 0; axesIndex < axes.size(); axesIndex++) {
```

Comment:
We don't use camel case for variable names.

Reply:
This is how it was originally in the file. I make sure not to introduce camel-casing for newly created variables.